python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -122,17 +122,23 @@ def __init__(self, op: PhysicalOperator, inqueues: List[Deque[MaybeRefBundle]]):
         self.num_completed_tasks = 0
         self.inputs_done_called = False
 
-    def initialize_progress_bars(self, index: int) -> int:
+    def initialize_progress_bars(self, index: int, verbose_progress: bool) -> int:
         """Create progress bars at the given index (line offset in console).
 
         For AllToAllOperator, zero or more sub progress bar would be created.
         Return the number of progress bars created for this operator.
         """
+        is_all_to_all = isinstance(self.op, AllToAllOperator)
+        # Only show 1:1 ops when in verbose progress mode.
+        enabled = verbose_progress or is_all_to_all
```

Comment:
why we want to show progress bar for all-to-all op by default, even not in verbose mode?

Reply:
The reason I decided to show it was because the overall progress bar would otherwise just show 0% until the entire operation is done (basically no progress reporting at all).

Btw, I don't think there is a strong downside to showing more progress bars, since the ipywidgets integration is pretty nice (see screenshot).